src/interfaces/pyo3/mod.rs

```diff
@@ -1,9 +1,11 @@
+#[macro_use]
+mod macros;
+mod py_abe_policy;
+mod py_cover_crypt;
+
 use crate::error::Error;
 use pyo3::{pymodule, types::PyModule, PyResult, Python};
 
-pub mod py_abe_policy;
-pub mod py_cover_crypt;
-
```

Comment:
I don't know why you moved these lines. If I read this file from top to bottom, I would expect the module declaration to come between the global imports (e.g. `use pyo3::{...}`) and the local imports (e.g. `use py_abe_policy::{...};`) like it used to be.

But this is not a real problem here. Feel free to ignore this comment.